Program.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System;
+
+namespace TextAdventure
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            {
+                Console.WriteLine("Welcome to the White House! I'm President Trump ( ^ ◡ ^)");
+                Console.WriteLine("You must be the new security gaurd, do you want a tour?");
+    
+                string response = Console.ReadLine();
+
+                if (response == "Yes")
```

Comment:
This is good and works well as a first pass.  Could this if-statement be written to handle more options, such as "yes", "YES", "yES", "YEs", etc.?